opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java

```diff
@@ -68,6 +68,11 @@ void configureWifiRrm(OvsdbClient ovsdbClient, OpensyncAPConfig opensyncApConfig
             if (elementRadioConfig == null || rfElementConfig == null) {
                 continue; // don't have a radio of this kind in the map
             }
+            boolean autoChannelSelection = rfElementConfig.getAutoChannelSelection();
+            int backupChannel = elementRadioConfig.getActiveBackupChannel(autoChannelSelection);
+            LOG.debug("configureWifiRadios autoChannelSelection {} activeBackupChannel {}",
+                    autoChannelSelection, backupChannel);
+            
             AutoOrManualValue probeResponseThresholdDb = null;
             AutoOrManualValue clientDisconnectThresholdDb = null;
             if (elementRadioConfig != null && rfElementConfig != null) {
```

Comment:
this check seems to be redundant - with what's happening on line 68. although it is not part of current changes...

Reply:
Believe it is leftover from the period when we didn't have the configuration maps for the radios in the config maps. There was a brief period when the auto provision was initially disabled for some APs and some of these values were coming up null. It is probably safe to remove it now.

Reply:
well, the way it is now is worse.
(original) lines 68-70 did the null checks, and I would say they were doing a good job at it.
(new) line 76 was redundant, and that's where I commented.
the way things are right now, (new) lines 69 and 70 can throw an NPE.

Reply:
I was puzzled for the change myself as well. Now I get it, thank you for clarification.